go.mod

```diff
@@ -2,16 +2,30 @@ module github.com/brokeyourbike/lets-go-chat
 
 go 1.17
 
-require golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
-
 require (
 	github.com/caarlos0/env/v6 v6.7.2
 	github.com/go-chi/chi/v5 v5.0.5
-	github.com/go-playground/locales v0.14.0 // indirect
-	github.com/go-playground/universal-translator v0.18.0 // indirect
 	github.com/go-playground/validator/v10 v10.9.0
 	github.com/google/uuid v1.3.0
+	golang.org/x/crypto v0.0.0-20211108221036-ceb1ce70b4fa
+	gorm.io/driver/postgres v1.2.1
+	gorm.io/gorm v1.22.2
+)
+
+require (
```

Comment:
please group require together
there is should be only one require 

I think there is no command for this, you have to change it manually